src/app/modules/core-bcp/components/core-practitioner-info/core-practitioner-info.component.ts

```diff
@@ -21,9 +21,6 @@ export class CorePractitionerInfoComponent {
 
   showValidationError: boolean;
   validationErrorMessage: string;
-
-  @Input()
-  public showFaxNumber: boolean;
   
   public items: CorePractitionerInfoFormItems;
```

Comment:
It was used in the practitioner info page, and toggled via the this flag to make the component reusable for facility and practitioner. However, since fax number got moved to the facility info page for practitioner, the flag is no longer needed since the fax number doesn't exist here for either forms.